exe/server/Dockerfile

```diff
@@ -1,17 +1,16 @@
-FROM golang:1.13.4 as builder
+FROM golang:1.13.7 as builder
+RUN apt-get update && apt-get install -y mesa-opencl-icd ocl-icd-opencl-dev
 RUN mkdir /app 
 WORKDIR /app 
-COPY go.mod go.sum ./
 COPY extern extern
+COPY go.mod go.sum ./
 RUN go mod download
 COPY . . 
-RUN GOOS=linux go build -o main exe/server/main.go
+RUN GOOS=linux go build -o main exe/server/main.go && \
+go run github.com/GeertJohan/go.rice/rice append --exec main -i ./build
 
-FROM alpine
-RUN apk --no-cache add ca-certificates wget && \
-    wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub && \
-    wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.28-r0/glibc-2.28-r0.apk && \
-    apk add glibc-2.28-r0.apk
+FROM ubuntu
```

Comment:
No more `alpine`. Tried to setup necessary dependencies but for one reason or another couldn't. 
TBH, also never compared how big was alpine after adding glibc... so maybe wasn't that minimal at all.
Anyway, maybe rooms for some MB improvements here, but is in the order of ~60MB.